hadoop-hdds/common/src/main/java/org/apache/hadoop/utils/db/DBStoreBuilder.java

```diff
@@ -220,4 +227,15 @@ private File getDBFile() throws IOException {
     return Paths.get(dbPath.toString(), dbname).toFile();
   }
 
+  private File getWALFile() throws IOException {
+    if (walPath == null) {
+      LOG.error("Write-ahead-log path is " +
```

Comment:
I would think that most people will never even add WAL as an option, so this should not be an error IMHO.